tests/snapshots/test_cli__test-project-suggest-json.snap

```diff
@@ -1449,7 +1449,7 @@ stdout:
       },
       {
         "name": "native-tls",
-        "notable_parents": "reqwest, hyper-tls, and tokio-native-tls",
+        "notable_parents": "hyper-tls and tokio-native-tls",
```

Comment:
I think this change is incorrect. Looking at the `reqwest` crate, the default enabled features are [`["default-tls"]`](https://github.com/seanmonstar/reqwest/blob/798df70fb0a4cd786ecc8cf770b33f1d56528524/Cargo.toml#L29), which enables [`"native-tls-crate"`](https://github.com/seanmonstar/reqwest/blob/798df70fb0a4cd786ecc8cf770b33f1d56528524/Cargo.toml#L33), which in turn turns on the dependency on this crate (https://github.com/seanmonstar/reqwest/blob/798df70fb0a4cd786ecc8cf770b33f1d56528524/Cargo.toml#L112). In the test crate we directly depend on `reqwest` with default features from the test crate (https://github.com/mozilla/cargo-vet/blob/668c5bbf4dc14b9d9cef74458dc98181374ec5cc/tests/test-project/Cargo.toml#L10), so that crate and those features should definitely be enabled, and `reqwest` should be considered to be a parent crate of `native-tls`.

I have a suspicion that these dependency edges aren't showing up because the code is incorrectly assuming that crate names match the corresponding optional dependency feature names, which is not necessarily the case.

Reply:
This does indeed look like a false positive.